modules/core/src/main/java/org/apache/ignite/internal/IgniteMXBeanImpl.java

```diff
@@ -330,8 +330,9 @@ public IgniteMXBeanImpl(IgniteKernal kernal) {
             ctx.state().changeGlobalState(newState, forceDeactivation, ctx.cluster().get()
                 .forServers().nodes(), false).get();
         }
-        catch (IgniteCheckedException e) {
-            throw U.convertException(e);
+        catch (IgniteException | IgniteCheckedException e) {
+            // Convert to deserializable one.
+            throw new RuntimeException("Unable to change cluster state to '" + state + "': " + e.getMessage());
```

Comment:
Why don't you use `throw U.jmException(e);` as other beans?

Reply:
Fixed